visualization/package.json

```diff
@@ -29,6 +29,9 @@
     "tsconfig.webpack.json",
     "cli.js"
   ],
+	"de_maibornwolff": {
```

Comment:
Why is the object called `de_maibornwolff`...? Isn't it rather something like `cc_json_file`, although I'm not super happy with that as well. What do you think?

Reply:
I put de_maibornwolff there since it was suggested here https://stackoverflow.com/questions/10065564/add-custom-metadata-or-config-to-package-json-is-it-valid that only one top-level key should be added, in which the custom keys are nested. However i have no strong opinion regarding this; we can also call it differently.

Reply:
do you think cc_json_file is better?

Reply:
I actually don't know..

Reply:
We decided to use the name `"codecharta" : { "apiVersion": "..." }`